src/app/containers/ArticleFigure/index.jsx

```diff
@@ -103,6 +104,7 @@ ArticleFigure.propTypes = {
   srcset: string,
   width: number.isRequired,
   showCopyright: bool,
+  service: string.isRequired,
```

Comment:
There is no need to have a `service` prop. This component uses a `Caption` which already gets `service` from `ServiceContext` https://github.com/bbc/simorgh/blob/efe9a3929e017d02a33f7ceac7940c40f465198a/src/app/containers/Caption/index.jsx#L59